infrastructure/plugin_creator.py

```diff
@@ -63,7 +63,7 @@
 
 
 def is_windows():
-    return "win" in sys.platform
+    return False if "darwin" in sys.platform else "win" in sys.platform
```

Comment:
Good idea! I think it would be little shorter to use:

```suggestion
    return "win" in sys.platform and "darwin" not in sys.platform
```